src/Microsoft.ML.Data/Prediction/IPredictionTransformer.cs

```diff
@@ -15,7 +15,6 @@ namespace Microsoft.ML.Runtime
     /// </summary>
     /// <typeparam name="TModel">The <see cref="IPredictor"/> used for the data transformation.</typeparam>
     public interface IPredictionTransformer<out TModel> : ITransformer
-        where TModel : IPredictor
```

Comment:
>where TModel : IPredictor [](start = 8, length = 25)

revert, not needed after all.

Reply:
it isneeded, since TModel cane an IPredictor or ICalibrator

---
In reply to: [238548671](https://github.com/dotnet/machinelearning/pull/1816#discussion_r238548671) [](ancestors = 238548671)